Model/src/main/java/org/eupathdb/common/model/contact/ContactUsParams.java

```diff
@@ -18,12 +19,14 @@ public ContactUsParams(
       String referrer,
       String[] ccEmails,
       String message,
+      String context,
```

Comment:
This update to the `ContactUsParams` constructor breaks two little-known unit tests. We should fix them before merging.

- [ ] Model/src/test/java/org/eupathdb/common/model/contact/ContactUsParamsTest.java
- [ ] Model/src/test/java/org/eupathdb/common/model/contact/ContactUsSubmitterTest.java

Reply:
Ah, I will fix this. When do these tests run?

Reply:
I don't believe these test are parts of our CI, so fixing them is definitely more of a nitpick.

Reply:
I just pushed a fix for this